coalib/settings/Setting.py

```diff
@@ -144,6 +152,29 @@ def __path_list__(self):
         """
         return [Setting.__path__(elem, self.origin) for elem in self]
 
+    def __url__(self):
+        """
+        Determines the url validity of this setting.
+
+        :return: url string
+        :raises ValueError: If the url is not valid.
```

Comment:
Please align docs to each other:
```
:return:            The url string.
:raises ValueError: If the ...
```
We still haven't changed to a new documentation format :/